package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "dependencies": {
+    "bcrypt": "^5.1.0",
+    "by": "^0.2.3",
+    "cors": "^2.8.5",
+    "dotenv": "^16.0.3",
+    "express": "^4.18.2",
+    "nodemon": "^2.0.20",
```

Comment:
nodemon is a dev dependency. Dev dependencies are packages needed only for developing the server but not in production. they are installed using the --save-dev or -D tag e.g
```
npm install -D nodemon
```